### DISCUSSION TITLE
一些微不足道的小建议

### DISCUSSION THREAD

**Discussion by binx6** [LibChecker/LibChecker#884] [Q&A] [answered]
在查看**库详情**的时候，由于一些`规则贡献者`撰写的**描述**冗长，导致有时看不完（看不到底），卡片又没法自由滑动，希望搞一个ScrollView或类似的东西解决一下，能增进体验感

希望`重载应用列表`这个功能加到异步线程里执行，在大量的应用情况下效率有点低。

规则仓库切换后有时会导致**库详情**打开失败，~~也可能是我的设备或网络原因~~，还有本身直接加载**库详情**也有点子慢

某些应用中的`权限`栏里的已经授予过的权限会被误标成**未授予**，目前见的最多的是**允许管理所有文件**（MANAGE_EXTERNAL_STORAGE）的误标问题，希望优化下检测手段

有些采取过特殊处理的**dex**，如**MT管理器**，~~不一定是dex大的问题哈~~，经过混淆，加载的时间过长，建议改善一下效率

暂时没别的了，我个人还是比较支持官方版本的🌟

**Comment by qhy040404** [comment]
1. 库详情确实有这个问题
2. 重载应用列表本来就是异步执行
3. GitHub本来国内的网络就不稳定，切换后几率打不开才是正常的，如果切换到GitLab打不开可能还有点问题
4. MANAGE_EXTERNAL_STORAGE对应的是授予所有文件的管理权限，这是一个特殊权限，和普通的读写手机存储不是一个权限
5. dex这一条我只能说pr-welcome

**Reply by su1216** [reply]
总结一下，只改1

**Reply by binx6** [reply, marked as answer]
> 总结一下，只改1 

5之前已得到过优化，然后2，3，4其实是我讲了废话（那个时候对安卓开发相关的有些东西还不是很熟，提了些无意义的）😂，1在上个礼拜已修复，所以这个讨论至此结束了

**Comment by binx6** [comment]
第4条我的锅，常规手段好像确实不行得先提权😂，不过`重载应用列表`的效率还是建议再想办法优化下，或者考虑写个jna/jni调用so来执行此操作

**Reply by qhy040404** [reply]
> 第4条我的锅，常规手段好像确实不行得先提权😂

不是不能获取权限状态，声明了这个权限的app不一定都去请求了这个权限
